.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  tox:
+    uses: ./.github/workflows/tox.yml
+    secrets: inherit
+  docker-build:
+    uses: ./.github/workflows/docker-build.yml
+    secrets: inherit
+  pypi:
+    needs: [tox, docker-build]
+    uses: ./.github/workflows/pypi.yml
+    secrets: inherit
+  docker-publish:
+    needs: [tox, docker-build]
+    uses: ./.github/workflows/docker-publish.yml
+    secrets: inherit
```

Comment:
We don't have `docker-publish` workflow for this repo. I realized we don't even have a `dockerfile` in this repo. I recommend removing anything that's docker related. It looks like it got removed from this [commit](https://github.com/aws-solutions-library-samples/osml-model-runner-test/commit/d50c4e55eb33282298f2c2307a50b189c8b28c6e)

And update the readme to remove those as well unless we are planning to bring this back in.

Reply:
Done